package pt.fabm.template.validation

class AuthException(cause:Throwable? = null) : Exception(cause = cause)
